EIPS/eip-6987.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 6987
```

Comment:
```suggestion
eip: 6988
```
EIP numbers are assigned by EIP editors and are generally the PR number which would make this EIP6988.  There isn't an issue or PR 6987 that I can see.